pkg/kubelet/eviction/helpers.go

```diff
@@ -23,15 +23,15 @@ import (
 	"strings"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/klog"
 	v1resource "k8s.io/kubernetes/pkg/api/v1/resource"
 	statsapi "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
 	evictionapi "k8s.io/kubernetes/pkg/kubelet/eviction/api"
 	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
-	schedulerutils "k8s.io/kubernetes/pkg/scheduler/util"
 	volumeutils "k8s.io/kubernetes/pkg/volume/util"
+	podinfo "k8s.io/kubernetes/pkg/api/pod"
```

Comment:
this is weird and confusing, podutil could be a better same as other packages import this with `podutil`